packages/lesswrong/lib/utils/componentUtils.ts

```diff
@@ -2,15 +2,15 @@
 export function debugShouldComponentUpdate(description: string, log: (message:string)=>void, oldProps: any, oldState: any, nextProps: any, nextState: any)
 {
   for(let key in nextState) {
-    if(oldState[key] != nextState[key]) {
+    if(oldState[key] !== nextState[key]) {
       log(`Updating ${description} because state.${key} changed`);
       return true;
     }
   }
 
   let changedProps: Array<any> = [];
   for(let key in nextProps) {
-    if(nextProps[key] != oldProps[key])
+    if(nextProps[key] !== oldProps[key])
```

Comment:
I'm actually not 100% sure this change wouldn't affect behavior, but it's in a bit of debugging logic that isn't used anywhere (since it requires `debugRerenders` which isn't currently set anywhere), so seems fine.